bigtable-hbase-1.x-parent/bigtable-hbase-1.x/pom.xml

```diff
@@ -60,7 +60,7 @@ limitations under the License.
     </dependency>
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
-      <artifactId>bigtable-client-core</artifactId>
+      <artifactId>bigtable-metrics-api</artifactId>
```

Comment:
why does metrics replace client core?

Reply:
I think this is a different change that also needs to be rebased.

Reply:
Nevermind: I think the classes that this module depended on were all moved from core into the metrics module

Reply:
?